src/components/DropdownSearch/index.js

```diff
@@ -101,6 +104,8 @@ function DropdownSearch({
   const handleClickSearch = () => {
     if (onClick) {
       onClick(countyCode);
+    } else if (!countyCode) {
+      router.push("/404");
```

Comment:
I think this should just be `else`

Reply:
@kilemensi I think we should check if there is query and `!countyCode`. Because what if a user clicks search icon while the query is empty.. I don't think we send them to 404